src/components/shared/button/index.tsx

```diff
@@ -1,21 +1,30 @@
 import React from 'react';
 import {Button as ButtonNativeBase} from 'native-base';
 import {GestureResponderEvent, StyleProp, ViewStyle} from 'react-native';
+import styles from './styles';
 
 interface Props {
   text: string;
   handleSubmit?: (event: GestureResponderEvent) => void;
-  style?: StyleProp<ViewStyle>;
+  customStyle?: StyleProp<ViewStyle>;
   isLoading?: boolean;
+  isDisabled?: boolean;
 }
 
-const Button = ({text, handleSubmit, style, isLoading}: Props) => {
+const Button = ({
+  text,
+  handleSubmit,
+  customStyle,
+  isLoading,
+  isDisabled,
+}: Props) => {
   return (
     <ButtonNativeBase
       onPress={handleSubmit}
       w={'50%'}
-      style={style}
-      isLoading={isLoading}>
+      style={[customStyle, styles.bottomButton]}
```

Comment:
customStyle should be the last element of the styling array, so it has precedence over any default styling
```suggestion
      style={[styles.bottomButton, customStyle]}
```

Reply:
Done.